.github/workflows/systest.yml

```diff
@@ -251,7 +254,7 @@ jobs:
           level: ${{ inputs.log_level }}
           clusters: 4
           norbac: 1
-        run: make -C systest run test_name=${{ inputs.test_name }}
+        run: make -C systest run test_name=${{ inputs.test_name }} gotestsum_fmt=${{ inputs.gotestsum_fmt }}
```

Comment:
For this to work the `Makefile` also needs to be updated 🙂

Reply:
I added it in the makefile and even tested it locally but somehow it vanished :D :thinking: . Thx for catching it